aasemble/django/apps/buildsvc/pkgbuild/python.py

```diff
@@ -41,7 +43,34 @@ def binary_pkg_name(self):
 
         return 'python-%s' % (pkgname,)
 
+    def get_pydist_overrides(self):
+        return self.get_aasemble_config().get('pydist_override', {})
+
+    def add_pydist_overrides(self):
+        override_dict = self.get_pydist_overrides()
+        if len(override_dict) == 0:
+            self.logger.info('No pydist-overrides in config')
+            return
+        pydist_overrides_str = (yaml.dump(override_dict, default_flow_style=False)).replace(':', '')
```

Comment:
Django templates let you iterate:

    {% for elem in somelist %}
    {{ elem.whatever }}
    {% endfor %}